apps/client/src/features/rundown/rundown-header/RundownHeader.module.scss

```diff
@@ -1,7 +1,8 @@
 .header {
   padding-inline: calc(2rem - 6px + 0.5rem) calc(1rem + 2px);
-  
+
   display: flex;
   align-items: center;
   justify-content: space-between;
+  column-gap: 0.5rem;
```

Comment:
in which case did you see the need for a column gap? I couldnt find a rundown small enough for it to be necessary?

and why using column-gap instead of gap since we expect no wrap?

Reply:
Resolved with https://github.com/cpvalente/ontime/pull/1228/commits/1fde566b5d537a76a7402efb7d9eaa3eefeba47a